charms/knative-eventing/tests/integration/test_charm.py

```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
These files were removed from the charm (#15) and are not going to be added in the future. Reason is we are not doing integration tests at a charm level, only at a bundle level. 
The reason why this file is here is because I haven't rebased.

Reply:
I believe having integration test for each charm is a bit overkill when we are already running bundle-level integration tests in the same repository on each PR. It seems like a duplication of effort and 2 places where we need to maintain tests. Also considering the way the unit test harness provides ways to tests events similar to integration tests I believe we can cover almost all of these charm specific checks there.

Reply:
@DomFleischmann and I discussed about this when writing `knative-operator`, decided to avoid integration tests at charm level. After Dom's reasoning, are there still concerns?